spec.md

```diff
@@ -388,13 +388,34 @@ both appear within the same JSON object.
   or application protocol's content-type metadata property. Normative rules
   for the binary mode and the content-type metadata mapping can be found 
   in the respective transport mapping specifications.
-    
+  
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### application-properties
+* Type: `Map`
+* Description: A list of "key-value" pairs providing information about the
+  event, which are used by an event-based application. The "key-value" pairs
+  MUST be in a flat structure. The value MUST be a string. There MUST be no
+  duplication of keys. One usage example: an application workflow can use a
+  "key-value" pair to correlate multiple events (from different event sources)
+  associated with a serverless application workflow.
+* Constraints:
+  * REQUIRED
```

Comment:
this should be OPTIONAL since not all events will carry additional information beyond the source, eventType, and eventId attributes (which are all REQUIRED)

(in many cases, the "source" attribute alone would provide sufficient information for correlating multiple events "from different event sources")

Reply:
@markfisher's comment is something I was going to bring up on the call but we ran out of time. @cathyhongzhang's your examples mentioned during the call really made it sound like you were providing more information about "the source", which made me wonder why the "source" attribute wouldn't provide (at least some of) this information?

Reply:
I am not sure if the source will for sure provide the needed info for correlation. But I agree that if the source does provide the info, the info can be used for correlating multiple events.